CodeEdit/Features/CEWorkspace/Models/CEWorkspaceFile.swift

```diff
@@ -113,6 +113,7 @@ final class CEWorkspaceFile: Codable, Comparable, Hashable, Identifiable, TabBar
     }
 
     func activateWatcher() -> Bool {
+        print("Activating watcher for \(self.url.description)")
```

Comment:
Does this print statement need to be there?

Reply:
nope, gonna delete all the random print statements now

Reply:
you can mark this as resolved